src/colvarscript.h

```diff
@@ -292,6 +293,14 @@ extern "C" {
            return COLVARS_OK;
            )
 
+  CVSCRIPT(cv_units,
+           "Get the current Colvars unit system",
+           0, 0,
+           { },
+           script->set_str_result(cvm::proxy->units);
```

Comment:
How about having a setter as well?

Reply:
I considered it, and there are arguments going both ways: because parsing a one line config string with the unit keyword does exactly the same job, and that's what the dashboard does now. So functionally it would be redundant, but it would be nice for the completeness of the scripting interface itself - which I think is important. So ok, I'm adding a setter.